test/tdigest_test.rb

```diff
@@ -145,4 +154,77 @@ def test_that_it_has_a_version_number
       tdigest.size.must_equal n
     end
   end
+
+  describe '#+' do
+    it 'works with empty tdigests' do
+      other = ::TDigest::TDigest.new(0.001, 50, 1.2)
+      (tdigest + other).centroids.size.must_equal 0
+    end
+
+    describe 'adding two tdigests' do
+      before do
+        @other = ::TDigest::TDigest.new(0.001, 50, 1.2)
+        [tdigest, @other].each do |td|
+          td.push(60, 100)
+          10.times { td.push(rand * 100) }
+        end
+      end
+
+      it 'has the parameters of the left argument (the calling tdigest)' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.